src/execution/execute.js

```diff
@@ -393,7 +393,13 @@ export function getOperationRootType(
 ): GraphQLObjectType {
   switch (operation.operation) {
     case 'query':
-      return schema.getQueryType();
+      const queryType = schema.getQueryType();
+      if (!queryType) {
+        throw new GraphQLError('Schema is not configured for queries', [
+          operation,
+        ]);
+      }
+      return queryType;
```

Comment:
The error message suggests that it's normal to have schema without query and it will work fine if you don't send `query` operations against it. I think error message should say that `query` type is mandatory.

Reply:
Yeah, I agree. I'll make that change